### PR TITLE
Change requirements/test.txt and add CHANGLOG.rst

### PR DESCRIPTION
**Pull request by stevenhua0320**
Merge these two files.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,6 @@
-====================
-Regolith Change Log
-====================
+=============
+Release Notes
+=============
 
 .. current developments
```
Comment (sbillinge) [inline CHANGELOG.rst:6]: Please add everything from the current changelog in here below the Release notes title

**Review by sbillinge — COMMENTED** [review]
please see inline comment.

File: CHANGELOG.rst (hunk 1 above)
Comment (stevenhua0320) [reply CHANGELOG.rst:6]: Hi Professor, I have verified that the current version of CHANGELOG.rst has everything from the old repo's CHANGELOG, you can see it by expand down the files in Files changed

Comment (sbillinge) [reply CHANGELOG.rst:6]: ok, thanks.  Sorry for not seeing that.

**Pull request merged**